t/api/15-search.t

```diff
@@ -78,15 +79,17 @@ subtest 'Job templates' => sub {
     my $group   = $schema->resultset('JobGroups')->create({name => 'Cool Group'});
     my $product = $schema->resultset('Products')
       ->create({name => 'Awesome Product', arch => 'arm64', distri => 'AwesomeOS', flavor => 'cherry'});
-    my $test_suite = $schema->resultset('TestSuites')->create({name => 'banana'});
-    my $machine    = $schema->resultset('Machines')->create({name => 'arm64', backend => 'qemu'});
-    $schema->resultset('JobTemplates')->create(
+    my $test_suite        = $schema->resultset('TestSuites');
```

Comment:
This may seem nitpicky but `$test_suite` is a confusing name for **the** resultset
```suggestion
    my $test_suites        = $schema->resultset('TestSuites');
```